tests/src/Endpoint/FileEndpointTest.php

```diff
@@ -30,7 +30,7 @@ class FileEndpointTest extends TestCase
      */
     public function testInvalidPermanentId(string $permanentId): void
     {
-        $this->expectExceptionObject(new \InvalidArgumentException('Invalid permanent ID format (it should be formatted like 2024/1234567).'));
+        $this->expectException(\InvalidArgumentException::class);
```

Comment:
This exception message is a fixed string, I guess it makes no sense to assert it.